README.md

```diff
@@ -47,6 +47,41 @@ IRC channels:
 
 [Assembly Testing Documentation](docs/AssemblyTests.md)
 
+## Python bindings
```

Comment:
can you add this as a doc in docs/ and link to it instead? that way it gets installed with the package.